kandinsky/fonts/rasterizer.c

```diff
@@ -123,12 +123,15 @@ int main(int argc, char * argv[]) {
   }
 
   int glyph_width = maxWidth-1;
+  if (glyph_width == 9) { glyph_width =  10; } /* This was made to avoid a problem, the ratio of the width by the height was not */
+  if (glyph_width == 8) { glyph_width =  7; }  /* adequate and the fonts couldn't be compiled, this is useless with other fonts */
```

Comment:
Maybe there can way a better way to do this. If the rasterizer won't compile the font, there is probably a reason. For example in the output :
![ItalicLargeFontExtended](https://user-images.githubusercontent.com/80424145/175832685-a07f6ed6-9418-4c8b-a5dc-f61776d7a457.png)
Some chars are out of their boxes. So maybe you should investigate instead of doing this type of fix that lead to a code that is very hard to understand.